alf/initializers.py

```diff
@@ -21,6 +22,15 @@
 import alf.utils.math_ops as math_ops
 
 
+def _is_elementwise_op(op):
+    """Check whether ``op`` is an elementwise operation."""
```

Comment:
By elementwise, do you include operations like normalizing e.g. sum-to-one normalize?

Reply:
Things like sum-to-one or softmax are not elementwise.

Reply:
> 

Then sum-to-one will not be correctly checked by this function? Because reshaping or not won't change the normalization result.